api/cloudscale.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright (c) 2019, VSHN AG, info@vshn.ch
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Generate deepcopy methodsets
+//go:generate controller-gen object:headerFile=../hack/boilerplate.go.txt paths=./...
+
+// Generate crossplane-runtime methodsets (resource.Claim, etc)
+//go:generate angryjet generate-methodsets --header-file=../hack/boilerplate.go.txt ./...
```

Comment:
https://github.com/crossplaneio/stack-rook/pull/21

You might be interested in the above pattern we arrived on recently to pin `controller-gen` and `angryjet` using Go modules.

Reply:
Nice, thanks